Dockerfile

```diff
@@ -20,7 +20,7 @@ run apt-get update && apt-get install --yes software-properties-common && \
     apt-get update && apt-get upgrade --yes && \
     apt-get install --yes \
         wget build-essential clang sed gdb clang-format git ssh shellcheck \
-        libc++-dev libc++abi-dev
+        libc++-dev libc++abi-dev python3 pip
```

Comment:
In addition to, or as an alternative to, `pip`, we could install the `yapf3` package. Then, in nginx-datadog's CI, we wouldn't need the [install python dependencies](https://github.com/DataDog/nginx-datadog/blob/6b36bb93287963479ca3cfe73e24a51846f7cd66/.circleci/config.yml#L127-L130) step.